src/workspaces.c

```diff
@@ -172,6 +173,28 @@ _osd_update(struct server *server)
 	}
 }
 
+/* cosmic workspace handlers */
+static void
```

Comment:
Consider creating a separate `workspace_create()` which `handle_workspace_create()` can call. Makes it easier to share code

Reply:
That commit (adding support for clients to create workspaces) is supposed to be deleted before we merge, it was only created to aid in testing of the sfwbar implementation. Most of the reason for me not wanting to implement that part in labwc is that it suddenly shares the responsibility of who configures workspaces with external clients and thus rc.xml doesn't show the full picture. We could consider allowing clients to create / remove workspaces at a later point though, the protocol side of the implementation is already there.